app/View/Attributes/add_attachment.ctp

```diff
@@ -68,8 +68,10 @@
             echo $this->Form->input('advanced', array(
                     'type' => 'checkbox',
                     'checked' => false,
```

Comment:
Please set `checked` to `true` and enable by default, most people want this feature all the time.
It's a comparable (default) feature to when all the hashes were taken automatically.

Reply:
added this